### PR TITLE
In addition, for the webapi under multiple namespaces, the display on…

### PR DESCRIPTION
**Pull request by Dean-ZhenYao-Wang**
… the page will be repeated,

Such as:

API/Orchard. AAA/Sto / {id}

API/Orchard. BBB/Sto / {id}

However, actually. Only API/Orchard. AAA/Sto/{id} is the correct result .

For example I solved the problem of just now, I defined a CustomApiExplorer, and use it to replace the system's own ApiExploer service, then the problem is solved,

### CODE REVIEW

File: App_Start/SwaggerConfig.cs [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+            public CachingSwaggerProvider(ISwaggerProvider swaggerProvider)
+            {
+                _swaggerProvider = swaggerProvider;
+            }
+
+            public SwaggerDocument GetSwagger(string rootUrl, string apiVersion)
+            {
+                var cacheKey = String.Format("{0}_{1}", rootUrl, apiVersion);
+                SwaggerDocument srcDoc = null;
+                //ֻ��ȡһ��
```
Comment (SzymonSel) [inline App_Start/SwaggerConfig.cs]: Please provide english comments only :)

**Comment by Dean-ZhenYao-Wang** [comment]
Commits on Oct 28, 2017

**Comment by SzymonSel** [comment]
Hi, could please remove unnecessary comments, keep the namespace and use the existing brackets style?

**Comment by SzymonSel** [comment]
Also, could you please limit your code, to only making it work with the given problem? We do not need xml comments and caching

**Comment by Dean-ZhenYao-Wang** [comment]
You can look at this project 
https://github.com/Dean-ZhenYao-Wang/Orchard_SimplifiedVersion

**Pull request closed without merging**